src/components/App/App.tsx

```diff
@@ -28,6 +29,7 @@ const App: FC = () => {
                     <UsernameInput label={"Email"} />
                     <UsernameInput label={"Phone number"} />
                     <UsernameInput label={"LinkedIn"} /> */}
+                    <ImgInput />
```

Comment:
Super! Pero se ve medio raro que los textfields tengan diferente tamaño, no? 

<img width="421" alt="Screen Shot 2023-04-22 at 12 22 28 PM" src="https://user-images.githubusercontent.com/52969178/233795647-a4b3784a-0315-4003-9250-52d55b2c8445.png">

Habrá alguna manera de que logremos que los _**TextField**_ components tengan el mismo tamaño? Algo así: 

<img width="490" alt="Screen Shot 2023-04-22 at 12 22 11 PM" src="https://user-images.githubusercontent.com/52969178/233795688-8b8807a6-c052-4fec-8add-7226ade1a2c3.png">
(check los props de both components)

Reply:
Alrighty the required changes have been implemented ;)